diff --git a/scripts/revert-proof-vitest-runner.mjs b/scripts/revert-proof-vitest-runner.mjs
index b600ad1..a9a59f1 100644
--- a/scripts/revert-proof-vitest-runner.mjs
+++ b/scripts/revert-proof-vitest-runner.mjs
@@ -113,7 +113,7 @@ export default class RevertProofVitestRunner extends TestRunner {
     try {
       await fn();
     } catch (err) {
-      verdict.set(test, isObject(err) && branded.has(err));
+      verdict.set(test, (isObject(err) && err.constructor === chai.AssertionError) || branded.has(err));
       throw err;
     }
   }
