diff --git a/scripts/revert-proof-vitest-runner.mjs b/scripts/revert-proof-vitest-runner.mjs
index d8bcf0d..b93b5c4 100644
--- a/scripts/revert-proof-vitest-runner.mjs
+++ b/scripts/revert-proof-vitest-runner.mjs
@@ -101,7 +101,7 @@ export default class RevertProofVitestRunner extends TestRunner {
     try {
       await fn();
     } catch (err) {
-      verdict.set(test, isObject(err) && branded.has(err));
+      verdict.set(test, err instanceof chai.AssertionError || (isObject(err) && branded.has(err)));
       throw err;
     }
   }
