--- a/web/src/graph/pack-mirror-gl.ts	2026-09-26 18:38:23.494464985 +0000
+++ b/web/src/graph/pack-mirror-gl.ts	2026-09-26 18:39:26.378254390 +0000
@@ -141,10 +141,8 @@
     if (fill && opts.letterbox) {
       paintLetterboxBarsThree(renderer, fill, dst, innerAbs, this.scratch.bars);
     }
-    if (this.material.map !== texture) {
-      this.material.map = texture;
-      this.material.needsUpdate = true;
-    }
+    this.material.map = texture;
+    this.material.needsUpdate = true;
     renderer.setScissorTest(true);
     renderer.setViewport(ix, iy, iw, ih);
     renderer.setScissor(ix, iy, iw, ih);
