diff --git a/web/src/ui/drawer-context.ts b/web/src/ui/drawer-context.ts
index 6293e0d..a580418 100644
--- a/web/src/ui/drawer-context.ts
+++ b/web/src/ui/drawer-context.ts
@@ -18,7 +18,7 @@ type DrawerContextBase = {
 
 export type NestCamsDrawerContext = DrawerContextBase & {
   spec: PluginView & { id: "nest-cams" };
-  devices: readonly Device[];
+  devices?: readonly Device[];
 };
 
 export type GenericDrawerContext = DrawerContextBase & {
